pkg/ebpf/tracer_windows.go

```diff
@@ -0,0 +1,83 @@
+// +build windows
+
+package ebpf
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"sync"
+)
+
+// CurrentKernelVersion is not implemented on non-linux systems
+func CurrentKernelVersion() (uint32, error) {
+	return 1, nil
+}
+
+type Tracer struct {
+	config      *Config
+	portMapping *PortMapping
```

Comment:
You can remove all these fields outside of `Config`. They're not doing anything yet and we're not even sure which ones we want to keep.